deepchecks/tabular/checks/distribution/train_test_feature_drift.py

```diff
@@ -218,10 +229,10 @@ def add_condition_drift_score_not_greater_than(self, max_allowed_psi_score: floa
 
         Parameters
         ----------
-        max_allowed_psi_score: float , default: 0.2
-            the max threshold for the PSI score
-        max_allowed_earth_movers_score: float , default: 0.1
-            the max threshold for the Earth Mover's Distance score
+        max_allowed_categorical_score: float , default: 0.2
+            the max threshold for the categorical drift score
+        max_allowed_numerical_score: float ,  default: 0.1
+            the max threshold for the numerical drift score
```

Comment:
it's not "numerical drift" and "categorical drift". it's "numeric variable drift score" and "categorical variable drift score"